### PR TITLE
Update for compatible with Redmine 4.0.5

### LINKED ISSUE #86: Not compatible with Redmine 4.x
Hi
I've recently upgraded to Redmine 4.0.1 and this plugin is not compatible with it (due to Rails incompatibility I think)
Any plans to fix it?
Thanks

### PR DESCRIPTION
**Pull request by RuslanRadko**
Tested on:
redmine 4.0.5, ruby 2.5.0, ruby 2.6.3, rails 5.2.3

Fixes #86.

### CODE REVIEW

File: init.rb [hunk 1]
```diff
@@ -9,16 +9,16 @@
   version '3.0.1'
   url 'http://stgeneral.github.io/redmine-progressive-projects-list/'
   author_url 'https://github.com/stgeneral'
-  requires_redmine :version_or_higher => '3.0'
+  requires_redmine version_or_higher: '3.0'
```
Comment (stgeneral) [inline init.rb]: 4.0

File: lib/progressive/projects_list_view_listener.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 class Progressive::ProjectsListViewListener < Redmine::Hook::ViewListener
-  render_on :view_layouts_base_sidebar, :partial => "progressive_sidebar"
-  render_on :view_projects_show_left,   :partial => "progressive_overview", :locals => {:side => :left}
-  render_on :view_projects_show_right,  :partial => "progressive_overview", :locals => {:side => :right}
+  render_on :view_layouts_base_sidebar, partial: 'progressive_sidebar'
+  render_on :view_projects_show_left,   partial: 'progressive_overview', locals: { side: :left }
+  render_on :view_projects_show_right,  partial: 'progressive_overview', locals: { side: :right }
 
-  def view_layouts_base_html_head(context)
-    stylesheet_link_tag('progressive_projects_list', :plugin => :progressive_projects_list)
```
Comment (stgeneral) [inline lib/progressive/projects_list_view_listener.rb]: No need for this change.

File: lib/progressive/project_decorator.rb [hunk 3]
```diff
@@ -37,9 +35,9 @@ def issues_progress(open)
 
   # Cloned from Version#completed_percent
   def issues_completed_percent
-    if issues.count == 0
+    if issues.count.zero?
```
Comment (stgeneral) [inline lib/progressive/project_decorator.rb]: Please, keep this code consistent with ‘Version#completed_percent’ method.

**Review by stgeneral — CHANGES_REQUESTED** [review]

File: lib/progressive/projects_list_view_listener.rb (hunk 2 above)
Comment (RuslanRadko) [reply lib/progressive/projects_list_view_listener.rb]: done

File: lib/progressive/project_decorator.rb (hunk 3 above)
Comment (RuslanRadko) [reply lib/progressive/project_decorator.rb]: done

**Pull request merged**